### ISSUE TITLE
Update docstrings for caer.color

### ISSUE THREAD

**Issue by jasmcaus** [jasmcaus/caer#13]

**Comment by MalikMAlna** [comment]
This still need some updating?

**Comment by jasmcaus** [comment]
No, not really. They were here as a reminder for me. I'm working on some update which should hopefully be pushed out today. After that, you can scout around for possible bugs. Thanks :)

**Comment by jasmcaus** [comment]
> This still need some updating?

While we don't require docstrings as of the moment, there are a couple of functions that require to be type-annotated (see #19). If you'd like to contribute to this, feel free to submit a PR.

Thanks!
-J
